x-pack/plugins/reporting/server/lib/tasks/execute_report.ts

```diff
@@ -221,6 +221,7 @@ export class ExecuteReportTask implements ReportingTask {
       docOutput.content_type = output.content_type || unknownMime;
       docOutput.max_size_reached = output.max_size_reached;
       docOutput.csv_contains_formulas = output.csv_contains_formulas;
+      docOutput.csv_rows = output.csv_rows;
```

Comment:
I think it is missing in `x-pack/plugins/reporting/server/lib/store/mapping.ts`.

Reply:
We have discussed moving the location `csv_rows` in the interface. Since we are not aggregating on the value yet, not having it in the mapping makes sense for now. :)